src-docs/src/components/guide_section/guide_section_parts/guide_section_code.tsx

```diff
@@ -16,10 +16,15 @@ export const GuideSectionExampleCode: FunctionComponent<GuideSectionExampleCode>
   code,
   type = GuideSectionTypes.JS,
 }) => {
+  const isJavascript = useMemo(
+    () => [GuideSectionTypes.JS, GuideSectionTypes.TSX].includes(type),
```

Comment:
I went with `.includes` in case we ever add another type in the future(??) but honestly

```suggestion
    () => type === GuideSectionTypes.JS || type === GuideSectionTypes.TSX,
```
type === GuideSectionTypes.JS || type === GuideSectionTypes.TSX,
```

is probably fine if we'd rather have that / think it's easier to read

Reply:
@constancecchen making suggestions on @constancecchen's PR is the good kind of inception 😆

Reply:
I like `includes` 👍